crates/wasm-encoder/src/core/code.rs

```diff
@@ -398,11 +396,8 @@ impl Encode for BlockType {
 impl TryFrom<wasmparser::BlockType> for BlockType {
     type Error = ();
```

Comment:
Given the nice `reencode::Error` I think it'd be reasonable to switch this from `()` to that instead.

Reply:
Also, while you're here, it might make sense to also move all the `From`/`TryFrom` impls into the `reencode` module too?

Reply:
Done